app/Services/CampaignService.php

```diff
@@ -68,6 +68,8 @@ public function getPostTotals($campaign)
                     DB::raw('SUM(status = "pending") as pending_count'),
                     DB::raw('SUM(status = "rejected") as rejected_count'))
                 ->whereIn('status', ['accepted', 'pending', 'rejected'])
+                // @TODO: if we ever want to expose other post types in Rogue to review/count, update this.
+                ->whereIn('type', ['photo', 'text'])
```

Comment:
I wonder if this makes sense to expose in a static `$reviewable` property on the post model?

Reply:
ooo yes I love this idea! Would this be something like [`scopeWithTag`](https://github.com/DoSomething/rogue/blob/master/app/Models/Post.php#L335-L340)?

Reply:
Oh yeah, I had been thinking just a public static property that you could grab here (like `Post::$reviewable`) but a query scope makes even better sense!